packages/magento-customer/components/OrderDetails/OrderDetails.tsx

```diff
@@ -105,255 +43,230 @@ export function OrderDetails(props: OrderDetailsProps) {
-              <div>{discount?.label}</div>
-              <div>
-                <Skeleton width={72} />
-              </div>
-            </TotalsRow>
-          ))}
-
-          <TotalsRow>
-            <div>Shipping</div>
-            <div>
-              <Skeleton width={72} />
-            </div>
-          </TotalsRow>
-
-          <TotalsDivider />
-
-          <TotalsVat>
-            <div>Total (incl. VAT)</div>
-            <div>
-              <Skeleton width={72} />
-            </div>
-          </TotalsVat>
-        </TotalsContainer>
-      </SectionContainer>
-    )
-  }
-
-  return (
-    <SectionContainer
-      labelLeft='Order details'
-      borderBottom
-      sx={[
-        (theme) => ({
-          '& .SectionHeader-root': {
-            marginTop: theme.spacings.sm,
-            marginBottom: theme.spacings.sm,
-          },
-        }),
-        ...(Array.isArray(sx) ? sx : [sx]),
-      ]}
-    >
-      <OrderDetailsInnerContainer>
-        <div>
-          <OrderDetailTitle>
-            <Trans id='Order number' />
-          </OrderDetailTitle>
-          <div>{number}</div>
-        </div>
-
-        <div>
-          <OrderDetailTitle>
-            <Trans id='Order status' />
-          </OrderDetailTitle>
-          <div>
-            Ordered: {order_date && dateFormatter.format(new Date(order_date))}
-            {/* Shipped */}
-          </div>
-        </div>
-
-        <div>
-          <OrderDetailTitle>
-            <Trans id='Shipping method' />
-          </OrderDetailTitle>
-          <div>
-            <div>{shipping_method ?? ''}</div>
+        <Box
+          className={classes.orderDetailContainer}
+          sx={[
+            (theme) => ({
+              gridColumnGap: theme.spacings.xxl,
+              gridRowGap: theme.spacings.md,
+              display: `grid`,
+              [theme.breakpoints.up('sm')]: {
+                gridTemplateColumns: `1fr 1fr`,
+                marginTop: theme.spacings.xxs,
+              },
+            }),
+            ...(Array.isArray(sx) ? sx : [sx]),
+          ]}
+        >
+          <Box className={classes.orderDetailRow}>
+            <SectionContainer
+              variantLeft='h5'
+              labelLeft={<Trans id='Order number' />}
+              className={classes.orderDetailTitle}
+              sx={{ '& .SectionHeader-root': { marginTop: 0, paddingBottom: '4px' } }}
+            />
+            <Typography>{number}</Typography>
+          </Box>
+
+          <Box className={classes.orderDetailRow}>
+            <SectionContainer
+              variantLeft='h5'
+              labelLeft={<Trans id='Order status' />}
+              className={classes.orderDetailTitle}
+              sx={{ '& .SectionHeader-root': { marginTop: 0, paddingBottom: '4px' } }}
+            />
+            <Typography>
+              <Trans id='Ordered items' />:{' '}
+              {order_date && dateFormatter.format(new Date(order_date))}
+            </Typography>
```

Comment:
The SectionContainer header is called 'Order status', we're not displaying the order status here, is this something we want?

Also shouldn't the content be inside the SectionContainer?

Reply:
Yes this should be inside the SectionContainer. 

In the old situatIon the order satus was the same, think it was misspeld, I changed it to the 'Order Date' now. The order status would be nice to implement here. I added it below the headertitle.